streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -470,7 +470,10 @@ public Subscription subscription(final Set<String> topics) {
                 for (final PartitionInfo partitionInfo : partitionInfoList) {
                     final TopicPartition partition = new TopicPartition(partitionInfo.topic(), partitionInfo.partition());
                     if (!allAssignedPartitions.contains(partition)) {
-                        log.warn("Partition {} is not assigned to any tasks: {}", partition, partitionsForTask);
+                        log.warn("Partition {} is not assigned to any tasks: {}"
+                                 + " Possible causes of a partition not getting assigned"
+                                 + " is that a topic defined in the topology has not been"
```

Comment:
`a` -> `another topic ... , resulting in no tasks created for this topology at all`.

Reply:
ack